src/Template/Vocabulary/add_sentences.ctp

```diff
@@ -55,6 +60,13 @@ $this->set('title_for_layout', $this->Pages->formatTitle($title));
             )
             ?>
         </div>
+        <div class="empty-info-text">
+            <?php
+                if ($vocabulary->count() == 0) {
+                    echo "There are no requests. ";
```

Comment:
This string include an unnecessary space at the end. It should also made translatable using `__("your translatable string here")`, otherwise it will stay in English regardless of the UI language.

Reply:
thanks, learned something new about php today!